hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/dc/MultiDatacenterRepositoryCommandExecutor.java

```diff
@@ -62,4 +71,17 @@ public MultiDatacenterRepositoryCommandExecutor(RepositoryManager repositoryMana
                     "' failed on DC '" + repoHolder.getDatacenterName() + "'.", e);
         }
     }
+
+    private <T> void handleExecutionRepositoryNotAvailableException(RepositoryCommand<T> command, boolean isRollbackEnabled, boolean shouldStopExecutionOnFailure, List<DatacenterBoundRepositoryHolder<T>> executedRepoHolders, DatacenterBoundRepositoryHolder<T> repoHolder, RepositoryNotAvailableException e) {
```

Comment:
In my opinion, extracting these two methods (`handleExecutionRepositoryNotAvailableException` and `handleExecutionException`) doesn't improve readability as they have many arguments and some of them are booleans. Maybe it would be better to stay with the inline version. WDYT?

Reply:
Ok, no problem - I've inlined them :)